randovania/game_connection/connector/prime_remote_connector.py

```diff
@@ -165,14 +165,14 @@ async def get_inventory(self) -> Inventory:
 
         return Inventory(inventory)
 
-    async def known_collected_locations(self) -> set[PickupIndex]:
-        """Fetches pickup indices that have been collected.
-        The list may return less than all collected locations, depending on implementation details.
-        This function also returns a list of remote patches that must be performed via `execute_remote_patches`.
+    async def check_for_collected_location(self) -> bool:
+        """
+        Queries the game for the multiworld magic item and checks if a new location was collected.
+        :return: True, if a new location was found and remote patches were executed. False otherwise.
         """
         multiworld_magic_item = self.multiworld_magic_item
         if multiworld_magic_item is None:
-            return set()
+            return False
```

Comment:
out of curiosity, when is this condition ever satisfied?

Reply:
Corruption used to not have this be None. Guess code got refactored around to being an exception?

Reply:
Gonna remove this so coverage is happy